search/controller/weedid/views.py

```diff
@@ -0,0 +1,18 @@
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+import requests
+
+
+def test(request):
+    return HttpResponse("TESTING VIEW")
+
+
+@csrf_exempt
+def elasticsearchQuery(request):
```

Comment:
Currently we have no convention for camelCase function names in Python code in the repo. Can we use snake case instead? `elasticsearch_query`?